diff --git a/src/corelib/gdi/fpg_gdi.pas b/src/corelib/gdi/fpg_gdi.pas
index 26d4515c..3a4a59e3 100644
--- a/src/corelib/gdi/fpg_gdi.pas
+++ b/src/corelib/gdi/fpg_gdi.pas
@@ -31,7 +31,6 @@
 {.$Define DEBUGKEYS}
 
 // enable or disable DND support. Disabled by default while implementing AlienWindows.
-{$define HAS_DND}
 
 {$IFDEF WINCE}
   // WinCE doesn't have DND support
@@ -3330,7 +3329,7 @@ initialization
   UnicodeEnabledOS := True;
   FontSmoothingType := DEFAULT_QUALITY;
 {$ELSE}
-  NeedToUnitialize := Succeeded(OleInitialize(nil));
+  NeedToUnitialize := False;
   WinVersion.dwOSVersionInfoSize := SizeOf(TOSVersionInfo);
   GetVersionEx(WinVersion);
   UnicodeEnabledOS := (WinVersion.dwPlatformID = VER_PLATFORM_WIN32_NT) or
@@ -3344,7 +3343,7 @@ initialization
 
 finalization
   if NeedToUnitialize then
-    OleUninitialize;
+    ;
 {$ENDIF}
 
 end.
